pkg/util/util.go

```diff
@@ -16,12 +16,23 @@
 package util
 
 import (
+	"cmp"
 	"fmt"
 	"os"
 	"path/filepath"
+	"slices"
 	"strings"
+
+	"golang.org/x/exp/maps"
 )
 
+// SortedMapKeys returns the sorted slice of map key strings.
+func SortedMapKeys[M ~map[K]V, K cmp.Ordered, V any](m M) []K {
```

Comment:
Need to update my Go knowledge. What is `~` used for?

Reply:
Full transparency, I copied and modified the signature of the maps.Keys function (https://pkg.go.dev/golang.org/x/exp/maps#Keys) and married with the signature of slices.Sort. I believe `~` means, "~T means the set of all types whose underlying type is T" --> https://www.calhoun.io/what-does-the-tilde-mean-in-go-generics/

Reply:
😵‍💫